pkg/controller/handler.go

```diff
@@ -227,6 +237,21 @@ func (p *Propeller) Handle(ctx context.Context, namespace, name string) error {
 		newWf, updateErr := p.wfStore.Update(ctx, mutatedWf, workflowstore.PriorityClassCritical)
 		if updateErr != nil {
 			t.Stop()
+			if workflowstore.IsWorkflowTooLarge(updateErr) {
```

Comment:
Should we move this logic to [here](https://github.com/flyteorg/flytepropeller/blob/master/pkg/controller/handler.go#L201-L202) since it's where we already check for update failures, ignore mutated workflows, increment counters... etc. ?

Reply:
we cannot, as the workflow is too large error comes from workflowstore

Reply:
but good point, i need to increment the counter

Reply:
Also I am not sure if we need to handle this as system error and increment it